include/ruby/ruby.h

```diff
@@ -1339,10 +1349,15 @@ int rb_big_sign(VALUE);
 #define OBJ_FROZEN(x) RB_OBJ_FROZEN(x)
 #define OBJ_FREEZE_RAW(x) RB_OBJ_FREEZE_RAW(x)
 #define OBJ_FREEZE(x) RB_OBJ_FREEZE(x)
+#define OBJ_WILL_FREEZE_RAW(x) RB_OBJ_WILL_FREEZE_RAW(x)
+#define OBJ_WILL_FREEZE(x) RB_OBJ_WILL_FREEZE(x)
+#define OBJ_CHILLED_RAW(x) RB_OBJ_CHILLED_RAW(x)
+#define OBJ_CHILLED(x) RB_OBJ_CHILLED(x)
```

Comment:
Since those are deprecated, I think we shouldn't add new macros without a `RB_` prefix.

Reply:
Thx, I was actually wondering what the prefix and not prefix thing was about.